bigbluebutton-html5/imports/api/chat/server/publishers.js

```diff
@@ -6,7 +6,7 @@ import { isAllowedTo } from '/imports/startup/server/userPermissions';
 
 Meteor.publish('chat', (credentials) => {
   if (!isAllowedTo('subscribeChat', credentials)) {
-    this.error(new Meteor.Error(402, "The user was not authorized to subscribe for 'chats'"));
+    throw new Meteor.Error(402, "The user was not authorized to subscribe for 'chats'");
```

Comment:
Theoretically we should use this.error. It is supposed to stop client's subscription.. but it was undefined (see the screenshot), that was strange.
http://docs.meteor.com/api/pubsub.html#Subscription-error
Maybe I should revert it back for now, until we figure why it's undefined.